src/App.js

```diff
@@ -1,210 +1,111 @@
-								 <h4><span>$400.00</span>$352.00</h4>
-							     <div className="button"><span><a href="singlepage.html">Read More</a></span></div>
-						    </div>
-							<div className="grid_1_of_5 images_1_of_5">
-								<img src="images/g7.png" />
-								 <h3>Lorem Ipsum is simply </h3>
-								 <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, in reprehenderit.</p>
-								  <h4><span>$300.00</span>$202.00</h4>
-							     <div className="button"><span><a href="singlepage.html">Read More</a></span></div>
-							</div>
-							<div className="grid_1_of_5 images_1_of_5">
-								 <img src="images/g8.png" />
-								 <h3>Lorem Ipsum is simply </h3>
-								 <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, in reprehenderit.</p>
-								 <h4><span>$400.00</span>$322.00</h4>
-							     <div className="button"><span><a href="singlepage.html">Read More</a></span></div>
-							</div>
-							<div className="grid_1_of_5 images_1_of_5">
-								 <img src="images/g1.jpg" />
-								 <h3>Lorem Ipsum is simply</h3>
-								 <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, in reprehenderit.</p>
-								 <h4><span>$700.00</span>$602.00</h4>
-							     <div className="button"><span><a href="singlepage.html">Read More</a></span></div>
-							</div>
-						</div>
-					</div>
-				</div>
-			</div>
-			<div className="clear"> </div>
-			<div className="footer">
-				<div className="section group">
-				<div className="col_1_of_4 span_1_of_4">
-					<h3>INFORMATION</h3>
-					<ul>
-						<li><a href="#">About us</a></li>
-						<li><a href="#">Sitemap</a></li>
-						<li><a href="#">Contact</a></li>
-						<li><a href="#">Terms and conditions</a></li>
-						<li><a href="#">Legal Notice</a></li>
-					</ul>
-				</div>
-				<div className="col_1_of_4 span_1_of_4">
-					<h3>OUR OFFERS</h3>
-					<ul>
-						<li><a href="#">New products</a></li>
-						<li><a href="#">top sellers</a></li>
-						<li><a href="#">Specials</a></li>
-						<li><a href="#">Products</a></li>
-						<li><a href="#">Comments</a></li>
-					</ul>
-				</div>
-				<div className="col_1_of_4 span_1_of_4">
-					<h3>YOUR ACCOUNT</h3>
-					<ul>
-						<li><a href="#">Your Account</a></li>
-						<li><a href="#">Personal info</a></li>
-						<li><a href="#">Prices</a></li>
-						<li><a href="#">Address</a></li>
-						<li><a href="#">Locations</a></li>
-					</ul>
-				</div>
-				<div className="col_1_of_4 span_1_of_4 footer-lastgrid">
-					<h3>Get in touch</h3>
-					<ul>
-						<li><a href="#"><img src="images/facebook.png" title="facebook" /></a></li>
-						<li><a href="#"><img src="images/twitter.png" title="Twiiter" /></a></li>
-						<li><a href="#"><img src="images/rss.png" title="Rss" /></a></li>
-						<li><a href="#"><img src="images/gpluse.png" title="Google+" /></a></li>
-					</ul>
-					<p>Design by <a href="#">W3layouts</a></p>
-				</div>
-			</div>
-			</div>
-		</div>
-	
+  let apples = props.products;
```

Comment:
Nice demonstration that you can name things whatever you want and it doesn't matter. Shows you understand how its actually working